### PR TITLE
fix dependency - sbom-utility

### LINKED ISSUE #19: Update dependency
github.com/mrutkows/sbom-utility doesn't see to redirect to github.com/CycloneDX/sbom-utility.

```bash
[...]
github.com/CycloneDX/license-scanner imports
        github.com/mrutkows/sbom-utility/log: github.com/mrutkows/sbom-utility@v0.0.0-20220322185037-eda8370b3803: invalid version: git ls-remote -q origin in /build/go/pkg/mod/cache/vcs/605d8e364eaf893408070ee2785bbcb1e639618895a0c4bc4a3fbdba869b8ff1: exit status 128:
        fatal: could not read Username for 'https://github.com': terminal prompts disabled
[...]
```

### PR DESCRIPTION
**Pull request by pritidesai**
Update dependency sbom-utility. Now importing utility from the CycloneDX org instead of personal github workspace.

Fixes #19

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -217,8 +224,8 @@ github.com/mitchellh/mapstructure v1.5.0/go.mod h1:bFUtVrKA4DC2yAKiSyO/QUcy7e+RR
 github.com/modern-go/concurrent v0.0.0-20180228061459-e0a39a4cb421/go.mod h1:6dJC0mAP4ikYIbvyc7fijjWJddQyLn8Ig3JB5CqoB9Q=
 github.com/modern-go/reflect2 v0.0.0-20180701023420-4b7aa43c6742/go.mod h1:bx2lNnkwVCuqBIxFjflWJWanXIb3RllmbCylyMrvgv0=
 github.com/modern-go/reflect2 v1.0.1/go.mod h1:bx2lNnkwVCuqBIxFjflWJWanXIb3RllmbCylyMrvgv0=
-github.com/mrutkows/sbom-utility v0.0.0-20220322185037-eda8370b3803 h1:YsCHY5tyoqLc8srkTxAltaQcSUjcQFFzRAI1XRsNJrI=
-github.com/mrutkows/sbom-utility v0.0.0-20220322185037-eda8370b3803/go.mod h1:QVtcKFO8qKRwNkQRSiM/MV71JbjoE9HqmzY4eZEv3tU=
+github.com/mrutkows/sbom-utility v0.0.0-20220615192221-fa8a20b2f5af h1:zMDFX0ckOPMg2Use+x+byix62iPiREe/TdBIMB/ZoIg=
```
Comment (markstur) [inline go.sum:227]: and hopefully this will go away after tidy

File: importer/importer_test.go [hunk 2]
```diff
@@ -420,7 +418,7 @@ func TestImporter_Validate(t *testing.T) {
 	t.Parallel()
 
 	// This test is particularly useful when DEBUG is set to show diff validate() errors. So force it on/off.
-	Logger.SetLevel(log.DEBUG)
+	Logger.SetLevel(4)
```
Comment (markstur) [inline importer/importer_test.go:421]: which Logger does this use?

**Review by markstur — COMMENTED** [review]
Can we remove more of the defunct logger?  See inline

File: importer/importer_test.go (hunk 2 above)
Comment (pritidesai) [reply importer/importer_test.go:421]: https://github.com/CycloneDX/sbom-utility/blob/86071171c03147419814decec536cd68bf05d48b/log/log.go#L47-L53

Comment (pritidesai) [reply importer/importer_test.go:421]: https://github.com/CycloneDX/license-scanner/blob/main/importer/importer.go#L21

**Review by markstur — APPROVED** [review]
LGTM

**Pull request merged**